diff --git a/src/main/java/com/vibrant_america/billing/models/Order/SkinCareSendKit.java b/src/main/java/com/vibrant_america/billing/models/Order/SkinCareSendKit.java
deleted file mode 100644
index 0d1d9d268..000000000
--- a/src/main/java/com/vibrant_america/billing/models/Order/SkinCareSendKit.java
+++ /dev/null
@@ -1,29 +0,0 @@
-package com.vibrant_america.billing.models.Order;
-
-import lombok.AllArgsConstructor;
-import lombok.Builder;
-import lombok.Data;
-import lombok.NoArgsConstructor;
-
-import java.util.List;
-import java.util.Map;
-
-@Data
-@Builder
-@NoArgsConstructor
-@AllArgsConstructor
-public class SkinCareSendKit {
-    private String operator;            //user
-    private String patient_name;
-    private String patient_street_address;
-    private String patient_city;
-    private String patient_state;
-    private String patient_zip_code;
-    private String patient_phone_number;
-    private String customer_country;            //US
-    private Map<String, Integer> phlebotomy_supplies;
-    private String shipping_method;       // [“First Overnight”, “Priority Overnight”, “Standard Overnight”, “Two Day AM”, “Two Day”, “Express Saver”, “Ground”, “In house”], // These are options, only one is required.
-    private String comments;
-    private String julien_barcode;
-
-}
diff --git a/src/main/java/com/vibrant_america/billing/service/service_implements/ProZOrderServiceImpl.java b/src/main/java/com/vibrant_america/billing/service/service_implements/ProZOrderServiceImpl.java
index 09ebd1f0a..6c2f72595 100644
--- a/src/main/java/com/vibrant_america/billing/service/service_implements/ProZOrderServiceImpl.java
+++ b/src/main/java/com/vibrant_america/billing/service/service_implements/ProZOrderServiceImpl.java
@@ -10,7 +10,6 @@ import com.vibrant_america.billing.mapper.master.PackagePriceDao;
 import com.vibrant_america.billing.models.GlobalConstants;
 import com.vibrant_america.billing.models.GoModels.Apply;
 import com.vibrant_america.billing.models.Order.ItemType;
-import com.vibrant_america.billing.models.Order.SkinCareSendKit;
 import com.vibrant_america.billing.models.lisCore.Customer;
 import com.vibrant_america.billing.models.lisCore.Patient;
 import com.vibrant_america.billing.service.service_interfaces.CustomerService;
@@ -106,49 +105,6 @@ public class ProZOrderServiceImpl implements ProZOrderService {
         return reList;
     }
 
-    public void sendSkinCareKit(String patientName, String patientStreetAddress, String patientCity,
-                                String patientState, String patientZipCode, String patientPhoneNumber,
-                                String customerCountry, Map<String, Integer> phlebotomySupplies, String shippingMethod,
-                                String comments, String julienBarcode, String auth,
-                                Integer sampleId
-    ) {
-        String url = "https://www.vibrant-america.com/lisapi/v1/lis/cloud-proxy/grpc/sendSkinPlacePatientOrders";
-        
-//        HttpHeaders headers = new HttpHeaders();
-
-        SkinCareSendKit skinCareSendKit = SkinCareSendKit.builder()
-                .operator("BillingSpring")
-                .patient_name(patientName)
-                .patient_street_address(patientStreetAddress)
-                .patient_city(patientCity)
-                .patient_state(patientState)
-                .patient_zip_code(patientZipCode)
-                .patient_phone_number(patientPhoneNumber)
-                .customer_country(customerCountry)
-                .phlebotomy_supplies(phlebotomySupplies)
-                .shipping_method(shippingMethod)
-                .comments(comments)
-                .julien_barcode(julienBarcode)
-                .build();
-
-        log.info("sendSkinCareKit parameters: " + new Gson().toJson(skinCareSendKit));
-
-        HttpHeaders headers = new HttpHeaders();
-        HttpEntity<SkinCareSendKit> requestEntity = new HttpEntity<>(skinCareSendKit, headers);
-        if (auth.contains("Bearer")) {
-            headers.add("Authorization", auth);
-        } else {
-            headers.add("Authorization", "Bearer " + auth);
-        }
-
-        ResponseEntity response = restTemplate.exchange(url, HttpMethod.POST, requestEntity, String.class);
-        if (response.getStatusCode().is2xxSuccessful()) {
-            log.info(sampleId + " sendSkinCareKit success");
-        } else {
-            log.info(sampleId + " sendSkinCareKit failed");
-        }
-    }
-
     public void sendEyeSerumKit(Integer sampleId, Long accessionId) {
         Map bodyMap = new HashMap();
         bodyMap.put("sample_id", sampleId.toString());
@@ -195,80 +151,6 @@ public class ProZOrderServiceImpl implements ProZOrderService {
             OrderTable orderTable = orderTableDao.selectByPrimaryKey(orderId);
             sendEyeSerumKit(orderTable.getSampleId(), orderTable.getJulienBarcode());
         }
-        
-
-        // Integer patientId = orderTable.getPatientId();
-
-        // Gson gson = new Gson();
-        // Patient lisCorePatient = customerService.retrievePatient(orderTable.getPatientId());
-
-        // Customer customer = customerService.fetchLisCoreCustomer(orderTable.getCustomerId());
-
-        // String patientName = lisCorePatient.getPatient_first_name() + " " + lisCorePatient.getPatient_last_name();
-        // String patientStreetAddress = "";
-        // String patientCity = "";
-        // String patientState = "";
-        // String patientZipCode = "";
-        // String patientPhoneNumber = "";
-        // String customerCountry = "US";
-        // String patientEmail = "";
-        // String comment = "";
-        // String julienBarcode = orderTable.getJulienBarcode().toString();
-        // try {
-        //     patientStreetAddress = lisCorePatient.getPatient_address().get(0).getStreet_address();
-        //     patientCity = lisCorePatient.getPatient_address().get(0).getCity();
-        //     patientState = lisCorePatient.getPatient_address().get(0).getState();
-        //     patientZipCode = lisCorePatient.getPatient_address().get(0).getZipcode();
-        //     customerCountry = customer.getCustomer_address().get(0).getCountry();
-        //     patientPhoneNumber = patientService.getPatientPhone(patientId);
-        //     patientEmail = patientService.getPatientEmail(patientId);
-        // } catch (Exception e) {
-        //     log.error(e.getStackTrace().toString());
-        // }
-
-        // //find the original token in orderTable
-        // String metaDataStr = new String(orderTable.getMetaData(), StandardCharsets.UTF_8);
-        // Map metaDataMap = gson.fromJson(metaDataStr, Map.class);
-        // String oldAuthorization = (String) metaDataMap.get("Authorization");
-
-        // try {
-        //     customerService.decodeJwtToken(oldAuthorization);
-        // } catch (Exception e) {
-        //     oldAuthorization = customerService.renewJwtToken(oldAuthorization);
-        // }
-        // //fetch previous order
-        // List<String> kitInfo = getSkinCarePreviousOrder(patientId, oldAuthorization);
-        // List<String> faceSerumKitList = filterSkinCareFaceSerumKitList(kitInfo);
-
-        // OrderDetailExample orderDetailExample = new OrderDetailExample();
-        // OrderDetailExample.Criteria orderDetailCriteria = orderDetailExample.createCriteria();
-        // orderDetailCriteria.andOrderIdEqualTo(orderId);
-        // List<OrderDetail> orderDetailList = orderDetailDao.selectByExample(orderDetailExample);
-
-        // Map<String, Integer> kitMap = new HashMap<>();
-        // for (OrderDetail orderDetail : orderDetailList) {
-        //     String itemType = (String) orderDetail.getItemType();
-        //     if (itemType.equals("packagePriceId")) {
-        //         Integer itemId = orderDetail.getItemId();
-        //         switch (itemId) {
-        //             case 801:
-        //                 kitMap.put("Eye Serum", 1);
-        //                 break;
-        //             case 803:
-        //                 kitMap.put("Skin Care Supplement", 1);
-        //                 kitMap.put("Eye Serum", 1);
-        //             case 802:
-        //                 for (String face : faceSerumKitList) {
-        //                     kitMap.put(face, 1);
-        //                 }
-        //                 break;
-        //         }
-        //     }
-        // }
-
-        // sendSkinCareKit(patientName, patientStreetAddress, patientCity, patientState, patientZipCode, patientPhoneNumber,
-        //         customerCountry, kitMap, "Two Day", comment, julienBarcode,
-        //         GlobalConstants.TokenAdmin.getValue(), orderTable.getSampleId());
     }
 
     @Override
diff --git a/src/main/java/com/vibrant_america/billing/service/service_interfaces/ProZOrderService.java b/src/main/java/com/vibrant_america/billing/service/service_interfaces/ProZOrderService.java
index 5ef994ca1..8b932c10b 100644
--- a/src/main/java/com/vibrant_america/billing/service/service_interfaces/ProZOrderService.java
+++ b/src/main/java/com/vibrant_america/billing/service/service_interfaces/ProZOrderService.java
@@ -11,12 +11,6 @@ public interface ProZOrderService {
 
     public List<String> filterSkinCareFaceSerumKitList(List<String> previousOrderKitList);
 
-    public void sendSkinCareKit(String patientName, String patientStreetAddress, String patientCity,
-                                String patientState, String patientZipCode, String patientPhoneNumber,
-                                String customerCountry, Map<String, Integer> phlebotomySupplies, String shippingMethod,
-                                String comments, String julienBarcode, String auth,
-                                Integer sampleId);
-
     public void sendSkinCareKitAsyncHelper(Long orderId) throws Exception;
 
     public List<Long> getProZPreviousOrder(Integer patientId);
